verticapy/performance/vertica/collection/profile_import.py

```diff
@@ -101,7 +170,7 @@ def _schema_exists_or_raise(self) -> None:
     def _tables_exist_or_raise(self) -> None:
         tables_in_schema = self._get_set_of_tables_in_schema()
         all_tables = getAllCollectionTables(
-            target_schema=self.target_schema, key=self.key
+            target_schema=self.target_schema, key=self.key, version=self.bundle_version
```

Comment:
The `getAllCollectionTables` function will throw an error when `self.bundle_version` is None. To avoid it, wouldn't be better to set it to `BundleVersion.LATEST`, like what is done in the `check_schema`?
What about changing `version` to  `BundleVersion.LATEST` when it is None inside `getAllCollectionTables`?

Reply:
I have set the bundle version to be defaulted to `BundleVersion.LATEST` in `ProfileImport`.